duktape/src/main/java/com/squareup/duktape/Duktape.java

```diff
@@ -69,20 +69,22 @@ public String evaluate(String script) {
    * defines the interface implemented by {@code object} that will be accessible to JavaScript.
    * {@code type} must be an interface that does not extend any other interfaces, and cannot define
    * any overloaded methods.
+   *
+   * Methods of the interface may return void or any of the following supported argument types:
+   * boolean, int, double, java.lang.Boolean, java.lang.Integer, java.lang.Double, java.lang.String.
```

Comment:
These should either be surrounded with `{@code ..}` or you can also use `{@link ..}` for them without the "java.lang." qualifier.